go-controller/pkg/ovn/multipolicy_test.go

```diff
@@ -616,8 +624,13 @@ var _ = ginkgo.Describe("OVN MultiNetworkPolicy Operations", func() {
 						gomega.Expect(err).NotTo(gomega.HaveOccurred())
 					}
 
-					ocInfo.asf.EventuallyExpectAddressSetWithAddresses(namespaceName1, []string{"10.1.1.1"})
-
+					if !expectMissingMappingFailure {
+						ocInfo.asf.EventuallyExpectAddressSetWithAddresses(namespaceName1, []string{"10.1.1.1"})
+					} else {
+						ocInfo.asf.EventuallyExpectAddressSetWithAddressesFailure(namespaceName1, []string{"10.1.1.1"})
+						gomega.Expect(fakeOvn.fakeRecorder.Events).To(gomega.HaveLen(1))
+						gomega.Expect(<-fakeOvn.fakeRecorder.Events).To(gomega.Equal(`Warning ErrorReconcilingPod failed to find bridge mapping for network: "network1", node "node1", ovn-bridge-mappings: ""`))
```

Comment:
OK, let's discuss the error msg here ...

```suggestion
						gomega.Expect(<-fakeOvn.fakeRecorder.Events).To(gomega.Equal(`Warning ErrorReconcilingPod failed to find bridge mapping for network: "network1" on node "node1"; Current ovn-bridge-mappings: ""`))
```

It's pretty much the same, but I think it reads better.